en/services/camera.md

```diff
@@ -27,29 +27,84 @@ These cameras have inbuilt support for MAVLink (but not necessarily camera proto
 - [PhaseOne Cameras](https://geospatial.phaseone.com/) ([source](https://geospatial.phaseone.com/resources-support/developer/mavlink/)).
 - [Workswell cameras](https://www.drone-thermal-camera.com/): WIRIS Pro, WIRIS Pro SC, WIRIS Security, WIRIS Agro, GIS-320 ([source](https://www.drone-thermal-camera.com/mavlink-interface-for-uav-cameras/)).
 
-## Camera Discovery/Connection
+### Camera Addressing
 
-Camera components are expected to follow the [Heartbeat/Connection Protocol](../services/heartbeat.md) and sent a constant flow of heartbeats (nominally at 1Hz).
+A vehicle can have both/either MAVLink cameras or non-MAVLink cameras that are connected to the autopilot, companion computer, or some other component.
+MAVLink cameras can be identified and addressed by their system and component id, but non-MAVLink cameras do not have a separate MAVLink identity and are "proxied" by component to which they are connected.
+
+In order to address (up to 6) non-MAVLink cameras proxied by an autopilot (or other component), these are allocated a camera id from `1` to `6` and can be uniquely addressed and identified using this id and the system id and component id of the proxying component.
+
+Camera messages include a camera id field, which will be set to the value of the camera id for messages originating from a non-MAVLink camera, or to `0` otherwise.
+Camera commands include a camera id field too, which is used for targetting:
+
+- non-MAVLink cameras when using the command protocol (or `0` for MAVLink cameras)
+- MAVLink cameras in missions using their component ID.
+- non-MAVLink cameras in missions attached to the autopilot.
+
+  > **Note:** There is no way to individually target proxied cameras in missions except for the those attached to the autopilot.
+  > All cameras attached to a companion computer can be addressed in missions using companion computer component id.
```

Comment:
As discussed, I would make this a general rule, not just for missions, to make it clear and discourage multiple cameras under one camera component ID which wasn't something possible before the recent device ID additions anyway.